http/sys_monitor.go

```diff
@@ -0,0 +1,57 @@
+package http
+
+import (
+	"fmt"
+	"net/http"
+
+	log "github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/vault/command/monitor"
+	"github.com/hashicorp/vault/vault"
+)
+
+func handleSysMonitor(core *vault.Core) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		ll := r.URL.Query().Get("log_level")
+		if ll == "" {
+			ll = "INFO"
+		}
+		logLevel := log.LevelFromString(ll)
+
+		if logLevel == log.NoLevel {
+			respondError(w, http.StatusBadRequest, fmt.Errorf("unknown log level"))
+			return
+		}
+
+		flusher, ok := w.(http.Flusher)
+		if !ok {
+			respondError(w, http.StatusBadRequest, fmt.Errorf("streaming not supported"))
+			return
+		}
+
+		isJson := core.SanitizedConfig()["log_format"] == "json"
```

Comment:
I think SanitizedConfig is really intended for the `vault debug` use case where we want a version of the config that doesn't contain sensitive information.  Can you just access the correct config field directly without using SanitizedConfig?

Reply:
Maybe?  I'll give that a shot.

Reply:
No, the field is not accessible directly, which is why I ended up going this route.  To avoid this though, I created a new `LogFormat()` function on `vault.Core`, which returns the required value.